test/TestHelper.cpp

```diff
@@ -234,7 +234,7 @@ void ImportTest::importState(json_spirit::mObject& _o, State& _state)
 	}
 }
 
-void ImportTest::importTransaction(json_spirit::mObject& _o)
+void ImportTest::importTransaction (json_spirit::mObject& _o, eth::Transaction& o_tr)
```

Comment:
the reason I did this is that according to style
static void importState(json_spirit::mObject& _o, eth::State& _state);
return value accepted as second parametr. 

that is beacause state variable is defined outside in test cpp's cause it requires some additional variables to be set.

State trueState(OverlayDB(State::openDB(td_stateDB_tmp.path())), BaseState::Empty, biGenesisBlock.coinbaseAddress);
importer.importState(o["pre"].get_obj(), trueState);

that is just how testing suites organized for now. not really the topic of this PR